cpp/src/Ice/PropertyNames.cpp

```diff
@@ -1,170 +1,160 @@
 // Copyright (c) ZeroC, Inc.
 
-// Generated by makeprops.py from PropertyNames.xml, Fri Sep 27 14:25:19 2024
+// Generated by makeprops.py from PropertyNames.xml, Tue Oct  1 15:07:27 2024
 
 // IMPORTANT: Do not edit this file -- any edits made here will be lost!
 
 #include "PropertyNames.h"
 
+const IceInternal::Property ProxyClassPropsData[] =
```

Comment:
New generated `ClassProps`

Reply:
I would generate a `using namespace IceInternal` and reduce qualifications.